ignite/distributed/comp_models/horovod.py

```diff
@@ -208,3 +207,6 @@ def barrier(self) -> None:
             # https://github.com/horovod/horovod/issues/159#issuecomment-424834603
             # hvd.allreduce(torch.tensor(0, device=self.device()), name="barrier")
             hvd.allreduce(torch.tensor(0, device="cpu"), name="barrier")
+
+        def _rank_not_in_group(self, group: Any) -> bool:
+            return group.included()
```

Comment:
Should not this be the opposite:
```suggestion
            return not group.included()
```